torchrl/trainers/helpers/losses.py

```diff
@@ -29,9 +29,10 @@
     SoftUpdate,
 )
 from torchrl.objectives.costs.common import LossModule
-from torchrl.objectives.costs.redq import REDQLoss
+from torchrl.objectives.costs.deprecated import REDQLoss_deprecated
```

Comment:
`REDQLoss` is memory intensive when used with large models. 
Some work is needed to make it less memory intensive.